app/lib/forem_stats_drivers/datadog_driver.rb

```diff
@@ -1,28 +1,32 @@
 require "httpclient"
+require "rest-client"
```

Comment:
`rest-client` is not a transitive dependency, so it needs to be required explicitly for the `c.use :rest_client` instruction to work